### ISSUE TITLE
reactiveui-events not working for platform projects possibly related to PackageReference style projects

### ISSUE THREAD

**Issue by PureWeen** [reactiveui/ReactiveUI#1526]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
I converted my projects all over to sdk style for shared projects and PackageReference for platform projects. After this was done (and compiling) I upgraded to the latest alpha bits of ReactiveUI and then all my ".events()" code stopped working

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**
https://github.com/PureWeen/ReactiveUiEvents

Compile the SLN and see the errors

**What is the expected behavior?**
.Events()  works in platform projects

**What is the motivation / use case for changing the behavior?**
.Events()  works in platform projects

**Which versions of ReactiveUI, and which platform / OS are affected by this issue? Did this work in previous versions of ReativeUI? Please also test with the [latest stable](https://www.nuget.org/profiles/reactiveui) and [development snapshot](https://reactiveui.net/docs/getting-started/installation/nuget-packages/development-package-feed)**

Does not work in 8.0.0-alpha0105
Works in 7.4.0

**Other information (e.g. stacktraces, related issues, suggestions how to fix)**

**Comment by ghuntley** [comment]
Are you using Visual Studio for Mac or Visual Studio for Windows (it matters!)

Looks to be okay on VS4Windows...

![image](https://user-images.githubusercontent.com/127353/31768833-c581fd5a-b51c-11e7-8e1f-d9e2c5d0aee0.png)

![image](https://user-images.githubusercontent.com/127353/31768842-cd175f74-b51c-11e7-9dc9-e7ca42ef8dbe.png)

**Comment by ghuntley** [comment]
Xamarin are working on official templates that use `PackageReference`; checking via backchannels if they are available for private preview and if so; I can share insights from them. Once I have these I'll be able to troubleshoot this further. For now - how did you convert your csproj?

**Comment by PureWeen** [comment]
- The included project wasn't created by me converting anything from the old format.  I have VS set to use Package Reference by default (Tools => options => Nuget Package Manager => General => Default package management format: PackageReference) so what you see here is just what VS creates

- I did a bad job of filling out questions when creating the ticket 👎   This all works fine in v7 but does not work in v8

- The issue specifically happens if you reference a second project that has the reactiveui-xamforms-events nuget in it as well.  In my included sample if you remove the reference to the netstandard project then the platform project compiles fine or if you uninstall the reactiveui-xamforms-events package from the netstandard project then it all compiles fine as well.

I pushed up another revision of my sample and you'll see a project called "FreshLonelyAndroidProject" that compiles fine

Then if you add the "AzureDeviceClient"  as a project reference to it it no longer compiles

in v7 the dll was named ReactiveUI.Events_XamForms.dll where as v8 it's ReactiveUI.Events.dll which is the same name as the platform one.

Are they colliding because of this?
